spec/rspec/mocks/partial_double_spec.rb

```diff
@@ -383,6 +383,22 @@ def defined_private_method
         object.implemented
       end
 
+      it 'wraps proxies in a reentrant mutex so we dont trigger deadlocks' do
```

Comment:
Also, I feel like this spec would better belong in `space_spec.rb`.  After all, that's where the bug was and the fix has nothing to do with partial doubles.